docs/llm/reference.md

```diff
@@ -84,11 +84,18 @@ The calculator supports the following `node_options` for tuning the pipeline con
 -    `optional bool dynamic_split_fuse` - use Dynamic Split Fuse token scheduling [default = true];
 -    `optional string device` - device to load models to. Supported values: "CPU" [default = "CPU"]
 -    `optional string plugin_config` - [OpenVINO device plugin configuration](https://docs.openvino.ai/2024/openvino-workflow/running-inference/inference-devices-and-modes.html). Should be provided in the same format for regular [models configuration](../parameters.md#model-configuration-options) [default = ""]
--    `optional uint32 best_of_limit` - max value of best_of parameter accepted by OVMS [default = 20];
--    `optional uint32 max_tokens_limit` - max value of max_tokens parameter accepted by OVMS [default = 4096];
+-    `optional uint32 best_of_limit` - max value of best_of parameter accepted by endpoint [default = 20];
+-    `optional uint32 max_tokens_limit` - max value of max_tokens parameter accepted by endpoint [default = 4096];
 
 
-The value of `cache_size` might have performance  implications. It is used for storing LLM model KV cache data. Adjust it based on your environment capabilities, model size and expected level of concurrency.
+The value of `cache_size` might have performance and stability implications. It is used for storing LLM model KV cache data. Adjust it based on your environment capabilities, model size and expected level of concurrency.
+You can track the actual usage of the cache in the server logs. You can observe in the logs output like below:
+```
+[2024-07-30 14:28:02.536][624][llm_executor][info][llm_executor.hpp:65] All requests: 50; Scheduled requests: 25; Cache usage 23.9%;
+```
+Consider increasing the `cache_size` parameter in case the logs report the usage getting close to 100%. When the cache is consumed, running requests might be terminated to get the cache released. 
```

Comment:
```suggestion
Consider increasing the `cache_size` parameter in case the logs report the usage getting close to 100%. When the cache is consumed, some of the running requests might be preempted to free cache for other requests to finish their generations (preemption will likely have negative impact on performance since preempted request cache will need to be recomputed when it get's processed again). When preemption is not possible i.e. `cache size` is very small and there is a single, long running request that consumes it all, then the request gets terminated when no more cache can be assigned to it, even before reaching stopping criteria. 
```